client/src/components/Error500/style.css

```
@@ -0,0 +1,3 @@
+.error500 {
+	width: 100%;
```

height is usually more concerning than width.
```suggestion
	width: 100%;
	height: 100%; or 100vh;
```

---

Adding a div didn't help so I just restyled it with view height